src/js/actions/generator.js

```diff
@@ -0,0 +1,70 @@
+import { SingleArticleItem, PluralArticleItem } from '../components/articleItem';
+import HtmlHelper from '../helpers/htmlHelper';
+import SourceItemFactory from '../components/sourceItem';
+
+//Strategy
+class Generator {
```

Comment:
There is no a strategy pattern

FYI:
[Strategy pattern](https://ru.wikipedia.org/wiki/%D0%A1%D1%82%D1%80%D0%B0%D1%82%D0%B5%D0%B3%D0%B8%D1%8F_(%D1%88%D0%B0%D0%B1%D0%BB%D0%BE%D0%BD_%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F)) take a look on javasctip implementation, please

Reply:
Done